hybrid/pkg/server/server_interceptor.go

```diff
@@ -0,0 +1,119 @@
+package hybrid_server
+
+import (
+	"context"
+
+	hclog "github.com/hashicorp/go-hclog"
+	nodeattestorv1 "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/server/nodeattestor/v1"
+)
+
+type ServerInterceptorInterface interface {
+	Recv() (*nodeattestorv1.AttestRequest, error)
```

Comment:
?

Reply:
instead of setting all functions as you are doing here,
you can rely on implementing the Interface you want.
So you dont need to create `Recv() (*nodeattestorv1.AttestRequest, error), Sent, etc)

Reply:
done